client/my-sites/sharing/connections/account-dialog-account.scss

```diff
@@ -47,8 +47,13 @@
 	left: 26px;
 }
 
-.account-dialog-account__name {
+.account-dialog-account__content {
 	display: inline-block;
 	padding-left: 48px;
 	color: $gray-dark;
 }
+
+.account-dialog-account__description {
+	font-weight: 300;
+	font-size: 90%;
```

Comment:
I don't think we want to use percent for font sizes.

Reply:
Why not?

Reply:
As far as I know we should only use [`px` or `em`](https://wordpress.com/design-handbook/typography/). Using percent makes the final size unpredictable, and probably not consistent with recommended sizes.

Reply:
Oh right, forgot about `em`  that's what I need here I think since I don't have the size of the font in the current context